src/app/static/choose-topic-page/topic-page.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, OnInit, ChangeDetectorRef } from '@angular/core';
+import { Tag } from 'src/app/models/Tag';
+import { TagService } from 'src/app/services/tags.service';
+import { Router } from '@angular/router';
+
+@Component({
+  selector: 'app-topic-page',
+  templateUrl: './topic-page.component.html',
+  styleUrls: ['./topic-page.component.css']
+})
+export class TopicPageComponent implements OnInit {
+
+  tag: Tag[];
+  tagName: string;
```

Comment:
What is this for?

Reply:
tagName deleted tag converted to tags for an array of tags